### PR TITLE
Validaiton with external logging

### PR DESCRIPTION
**Pull request by another-rex**
Provides some basic feedback publicly about what errors out importer is getting when attempting to parse the osv entries it's trying to import. 

Also added export PIPENV_IGNORE_VIRTUALENVS=1 to the run_tests.sh scripts to make sure the correct environment is used when running tests.

### CODE REVIEW

File: docker/importer/importer.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 import osv
 
 DEFAULT_WORK_DIR = '/work'
+PUBLIC_LOGGING_BUCKET = 'public-import-logging'
```
Comment (oliverchang) [inline docker/importer/importer.py:34]: nit: `osv-public-import-logging` to make it clearer?

File: docker/importer/importer.py [hunk 2]
```diff
@@ -356,6 +373,10 @@ def convert_blob_to_vuln(blob_name) -> Optional[Tuple[str, str]]:
           return None
         except Exception as e:
           logging.error('Failed to parse vulnerability %s: %s', blob_name, e)
+          # Don't include error stack trace as that might leak sensitive info
+          # List.append() is atomic and threadsafe, wow
```
Comment (oliverchang) [inline docker/importer/importer.py:377]: nit: remove the ", wow" part :)

File: docker/importer/importer.py [hunk 3]
```diff
@@ -296,13 +307,17 @@ def _process_updates_git(self, source_repo: osv.SourceRepository):
         logging.info('Entry does not have an OSV entry: %s', changed_entry)
       except Exception as e:
         logging.error('Failed to parse %s: %s', changed_entry, str(e))
+        # Don't include error stack trace as that might leak sensitive info
+        import_failure_logs.append('Failed to parse vulnerability ' + path)
```
Comment (oliverchang) [inline docker/importer/importer.py:311]: nit: this is inconsistent with below, which adds quotation marks.

**Review by oliverchang — COMMENTED** [review]
nice! 

Cloudbuild is failing though, with:

```
Step #0: Already have image (with digest): gcr.io/cloud-builders/git
Step #0: Submodule 'docker/worker/osv-test' (https://github.com/oliverchang/osv-test.git) registered for path 'osv-test'
Step #0: Submodule 'gcp/api/googleapis' (https://github.com/googleapis/googleapis) registered for path '../../gcp/api/googleapis'
Step #0: fatal: No url found for submodule path '../../osv-schema' in .gitmodules
```

File: docker/importer/importer.py (hunk 1 above)
Comment (another-rex) [reply docker/importer/importer.py]: Changed it to `osv-public-import-logs`

**Review by oliverchang — APPROVED** [review]

**Pull request merged**